picturae_importer.py

```diff
@@ -947,7 +947,12 @@ def run_all_methods(self):
         # unlocking database
 
         sql = f"""UPDATE mysql.user
-                 SET account_locked = 'n'
-                 WHERE user != '{picturae_config.USER}' AND host = '%';"""
+                  SET account_locked = 'n'
+                  WHERE user != '{picturae_config.USER}' AND host = '%';"""
 
         self.sql_csv_tools.insert_table_record(logger_int=self.logger, sql=sql)
+
+        value_list = [len(self.new_taxa), self.records_dropped]
+
+        create_monitoring_report(value_list=value_list, batch_size=self.batch_size, batch_md5=self.batch_md5,
```

Comment:
the create_monitoring_report function at the end of picturae_importer, which takes a custom value list as an argument. Let me know if there is a more efficient way to do this. Because having a value list here and a custom terms list in the config, seems kind of jerry rigged.